packages/eas-build-job/src/common.ts

```diff
@@ -210,6 +210,11 @@ export const StaticWorkflowInterpolationContextZ = z.object({
               id: z.string(),
             })
             .optional(),
+          pull_request: z
+            .object({
+              number: z.number(),
+            })
+            .optional(),
```

Comment:
```suggestion
            .optional(),
          number: z.number().optional(),
```
https://exponent-internal.slack.com/files/U8565SVQX/F09C292EPB2/zrzut_ekranu_2025-08-20_o_18.50.04.png